using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -1,150 +1,271 @@
-# Adobe Campaign Standard API reference
+# Campaign API reference
```

Comment:
Should we update the name in the Summary.md too?

Reply:
yes, will update there as well

Reply:
actually, will update in the api reference doc to match the summary